app/utils/url.js

```diff
@@ -7,7 +7,7 @@ export function isValidUrl(url = '') {
 }
 
 export function stripTrailingSlashes(url = '') {
-    return url.replace(/\/+$/, '').trim();
+    return url.trim().replace(/\/+$/, '').trim();
```

Comment:
I don't think we need the double trim here. Just the one before the replace should be good enough